### PR TITLE
SICPy: Chapter 1.0-1.1

### PR DESCRIPTION
**Pull request by Fidget-Spinner**
Some stuff up for discussion.

### CODE REVIEW

File: xml/chapter1/chapter1.xml [hunk 1]
```diff
@@ -335,28 +321,19 @@
...
-	first edition of the ECMAScript standard was led by Guy Lewis 
-	Steele Jr.<SPACE/>and completed in June 1997
-	(<CITATION>ECMA 1997</CITATION>).
-	The sixth edition, known as ECMAScript 2015, was led by 
-	Allen Wirfs-Brock and adopted by the General Assembly of ECMA in
-	June 2015 (<CITATION>ECMA 2015</CITATION>). 
+	by the Python interpreter.
+	<INDEX>Python interpreter, interpreting JavaScript</INDEX>
+	Python was a product of community-driven development.
+  After van Rossum's initial implementation at CWI, Python has been updated
```
Comment (Fidget-Spinner) [inline xml/chapter1/chapter1.xml:332]:
```suggestion
  After Van Rossum's initial implementation at CWI, Python has been updated
```

File: xml/chapter1/chapter1.xml [hunk 2]
```diff
@@ -227,27 +223,17 @@
...
-        was used as the programming language for the original version of this
-	book. From Scheme, JavaScript inherited its most fundamental design
-	principles,
-        such as lexically scoped first-class 
-        functions and dynamic typing.
+	in the Python language.
+        <INDEX>Van Rossum, Guido</INDEX>
+	The first Python interpreter was implemented by van Rossum
+        at <EM>Centrum voor Wiskunde en Informatica</EM> (CWI) for the Amoeba
+        operating system.
```
Comment (Fidget-Spinner) [inline xml/chapter1/chapter1.xml:234]: I should reference Guido's blog.

File: xml/chapter1/section1/subsection1.xml [hunk 3]
```diff
@@ -510,17 +501,10 @@
...
-	<INDEX>Wilde, Oscar (Perlis<APOS/>s paraphrase of)</INDEX>
-	<INDEX>value<SUBINDEX><ORDER>expression</ORDER>of an expression</SUBINDEX></INDEX>
-	<INDEX>Lisp<SUBINDEX>efficiency of</SUBINDEX></INDEX>
-	statement has a value (see exercise<SPACE/><REF NAME="ex:value_producing"/>).
-	This convention, together with the
-	reputation of JavaScript programmers as not caring about efficiency, leads
-	us to paraphrase a quip on Lisp programmers by Alan Perlis (who was himself
-	paraphrasing Oscar Wilde): <EM>JavaScript programmers know the value of
-	everything but the cost of nothing.</EM></FOOTNOTE>
+	statement.<FOOTNOTE>Expression statements in Python produce values in
```
Comment (martin-henz) [inline xml/chapter1/section1/subsection1.xml:504]: Good call to get rid of the Oscar Wilde reference in this footnote.

**Review by martin-henz — CHANGES_REQUESTED** [review]

**Comment by Fidget-Spinner** [comment]
For some reason, there's mixed tabs and spaces in the document. I've tried to keep consistent with the rest of the surrounding text whenever that happens.

File: xml/chapter1/chapter1.xml [hunk 4]
```diff
@@ -180,8 +180,8 @@
         <INDEX>Van Rossum, Guido</INDEX>
         Guido van Rossum.
         <INDEX>Monty Python</INDEX>
-        The name <QUOTE>Python</QUOTE> is a reference to Monty Python and is
-        a registered trademark of the Python Software Foundation.
+        The name <QUOTE>Python</QUOTE> is a reference to <EM>Monty Python</EM>
```
Comment (martin-henz) [inline xml/chapter1/chapter1.xml:183]: I think you can't expect "Monty Python" to be known across generations. It's already a bit "vintage". So I suggest:

..is a reference to the British comedy troupe < EM >Monty Python< /EM >...

**Review by martin-henz — CHANGES_REQUESTED** [review]

File: xml/chapter1/chapter1.xml [hunk 5]
```diff
@@ -335,28 +321,19 @@
...
-	The sixth edition, known as ECMAScript 2015, was led by 
-	Allen Wirfs-Brock and adopted by the General Assembly of ECMA in
-	June 2015 (<CITATION>ECMA 2015</CITATION>). 
+        In contrast with other
+        languages like Java and C, which usually
+        employ compilation to lower-level
+        languages, Python programs were initially
+        <EM>interpreted</EM>
+        by the Python interpreter.
+        <INDEX>Python interpreter, interpreting JavaScript</INDEX>
```
Comment (Fidget-Spinner) [inline xml/chapter1/chapter1.xml:330]:
```suggestion
        <INDEX>Python interpreter, interpreting Python</INDEX>
```

**Comment by Fidget-Spinner** [comment]
@martin-henz would it be alright if I merge the PR?

**Review by martin-henz — APPROVED** [review]

**Pull request merged**